src/Transports/MassTransit.ActiveMqTransport/ActiveMqBusFactory.cs

```diff
@@ -20,9 +20,12 @@ public static class ActiveMqBusFactory
         /// <returns></returns>
         public static IBusControl Create(Action<IActiveMqBusFactoryConfigurator> configure)
         {
-            var topologyConfiguration = new ActiveMqTopologyConfiguration(MessageTopology);
+            var topologyConfiguration = new ActiveMqTopologyConfiguration(MessageTopology, null);
+
             var busConfiguration = new ActiveMqBusConfiguration(topologyConfiguration);
 
+            topologyConfiguration.BusConfiguration = busConfiguration;

```

Comment:
Circular dependency, so must be refactored.

Reply:
new commits have different implementation